imports/api/questions.js

```diff
@@ -252,14 +252,15 @@ Meteor.methods({
    * @param {Question} question
    * @returns {Question} new question
    */
-  'questions.insert' (question) {
+  'questions.insert' (question, courseId) {
```

Comment:
since courseId is a property of question, it should be a separate argument. E.g. what if the question being inserted already has that property (e.g. from a copy or something)...